types/mapbox-gl/mapbox-gl-tests.ts

```diff
@@ -897,6 +897,13 @@ expectType<mapboxgl.Map>(
         expectType<undefined>(ev.originalEvent);
     }),
 );
+expectType<mapboxgl.Map>(
```

Comment:
Let's use $ExpectType for new tests. That's a de-facto standard. expectType<> is a thing from the past.